src/is_face_detector/main.py

```diff
@@ -0,0 +1,35 @@
+from utils import load_options, get_np_image, get_pb_image
+from is_wire.core import Logger, Channel, Subscription, Message
+from faceDetection import FaceDetector
+from is_msgs.image_pb2 import Image
+import cv2
+import time
+import re
+from options_pb2 import FaceDetectorOptions 
+
+def main():
+
+    service_name = "FaceDetector.Detection"
+    log = Logger(name=service_name)
+    op = load_options()
+    k = FaceDetector()
```

Comment:
avoid to use single letters to name variables. Choose a name that makes your code more readable